### ISSUE TITLE
Incorrect dependency on apollo-server-module-graphiql

### ISSUE THREAD

**Issue by Vadskye** [apollographql/apollo-server#608]
The package.json specifies a dependency on https://www.npmjs.com/package/apollo-server-module-graphiql with a version of v1.1.4. However, that package has not officially released a v1.1.4; its most recent version is v1.1.3. This breaks installation with `npm install`.

**Comment by tylerjbainbridge** [comment]
Having the same issue here.

**Comment by martijnwalraven** [comment]
Hmmm, I'm not sure what went wrong there. I thought I released v1.1.5 of all packages. Let me try again.

**Comment by martijnwalraven** [comment]
I had to release a new v1.1.6, but I think that should fix it.

**Comment by Vadskye** [comment]
Worked for me. Thanks for the quick fix!

**Comment by tylerjbainbridge** [comment]
Worked for me too! Thanks :)
